integr8ly/tutorial-web-app
# fix(dependencies): Update various dependencies

Replace `postcss-next` with `postcss-preset-env`; update `stylelint`, `stylelint-config-recommended-scss`, `stylelint-config-standard`, `stylelint-order`, and `stylelint-scss`.

## Motivation
Keep dependencies up to date.

## What
Update and/or replace dependencies related to SCSS/CSS processing.

## Why
Outdated dependencies - one of which was not longer supported.

## How
Increase version numbers to match the latest releases.

## Verification Steps

1. Pull down this PR's branch
2. Run `yarn install` to get the latest updates
3. Run `yarn build` to verify that the build passes
4. Run `yarn test` to verify that all tests pass
5. Start a local instance to verify that the UI builds/runs correctly.

## Checklist:


## Progress


## Additional Notes

Related to issue https://github.com/integr8ly/tutorial-web-app/issues/476
